### PR TITLE
example8

### PR DESCRIPTION
**Pull request by HesamShaelaie**

### CODE REVIEW

**Comment by odow** [comment]
Looks like we might have to relax the tolerances even further in the other tests. @joaquimg do you often see this issue with Ipopt?

File: docs/src/examples/DTMP_example1.jl [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# # Foundations of Bilevel Programming: Example 8
+# This example is from the book Decomposition Techniques in Mathematical Programming
+# Chapter 7.2, page 281, [url](https://www.springer.com/gp/book/9783540276852)
+
+# Bold point: using upperonly and loweronly, using equality in constraints
+
+# Model of the problem
+# First level
+# ```math
+
```
Comment (joaquimg) [inline docs/src/examples/DTMP_example1.jl:10]: remove space

File: docs/src/examples/DTMP_example1.jl [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# # Foundations of Bilevel Programming: Example 8
+# This example is from the book Decomposition Techniques in Mathematical Programming
+# Chapter 7.2, page 281, [url](https://www.springer.com/gp/book/9783540276852)
+
+# Bold point: using upperonly and loweronly, using equality in constraints
```
Comment (joaquimg) [inline docs/src/examples/DTMP_example1.jl:5]: we should point out that: upper/loweronly creates variable only in the proper model and hence might generate slightly smaller reformulated models

File: docs/src/examples/DTMP_example1.jl [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+using BilevelJuMP
+using Ipopt
+using JuMP
+using Test
+
+model = BilevelModel(Ipopt.Optimizer, mode = BilevelJuMP.ProductMode(1e-9))
+
+# Global variables
+atol = 1e-3
```
Comment (joaquimg) [inline docs/src/examples/DTMP_example1.jl:36]: lets remove this and add it explicitly in the `@test` lines

**Review by odow — APPROVED** [review]

**Pull request merged**